src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php

```diff
@@ -120,6 +121,21 @@ public function process(ContainerBuilder $container)
                 $args = [];
                 foreach ($parameters as $p) {
                     /** @var \ReflectionParameter $p */
+                    if (class_exists(Autowire::class) && $attr = $p->getAttributes(Autowire::class)[0] ?? null) {
```

Comment:
reading attributes should happen last: precedence should go from most explicit to less explicit, and most explicit is always hand written config (attributes on the DI tag, etc)

Reply:
I'm... a bit lost here.

Reply:
this if should go just before the one on L164/180 that reads as `if ($type &&...`

Reply:
Thanks :)